From 0d3af750a2ab1a219fc0fddbb48af55c75615327 Mon Sep 17 00:00:00 2001
From: Jacky Yuen <jackyyuen10@gmail.com>
Date: Thu, 15 Jan 2026 15:20:49 +0800
Subject: [PATCH 29/31] sunxi g2d driver-patch

Signed-off-by: Jacky Yuen <jackyyuen10@gmail.com>
---
 include/linux/g2d_driver.h | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/include/linux/g2d_driver.h b/include/linux/g2d_driver.h
index db038d364..0bc4063ee 100644
--- a/include/linux/g2d_driver.h
+++ b/include/linux/g2d_driver.h
@@ -320,6 +320,11 @@ enum color_range {
 	COLOR_RANGE_16_235 = 1,
 };
 
+typedef struct {
+	__u32 w;
+	__u32 h;
+} g2d_size;
+
 /* image struct */
 typedef struct {
 	int		 bbuff;
@@ -332,6 +337,7 @@ typedef struct {
 	__u32		 align[3];
 
 	g2d_rect	 clip_rect;
+	g2d_size	 resize;
 	g2d_coor	 coor;
 
 	g2d_color_gmt	 gamut;
@@ -527,4 +533,3 @@ typedef enum {
 } g2d_cmd;
 
 #endif	/* __G2D_DRIVER_H */
-
-- 
2.34.1

